core/src/context.rs

```diff
@@ -249,29 +251,7 @@ impl Context {
     }
 
     pub fn sign(&self, payload: String) -> Result<String, HolochainError> {
-        let handler = self.conductor_api.write().unwrap();
-        let request = format!(
-            r#"{{"jsonrpc": "2.0", "method": "agent/sign", "params": {{"payload": "{}"}}, "id": "{}"}}"#,
-            payload, ProcessUniqueId::new()
-        );
-
-        let response = handler
-            .handle_request_sync(&request)
-            .ok_or("Conductor sign call failed".to_string())?;
-
-        let response = JsonRpc::parse(&response)?;
-
-        match response {
-            JsonRpc::Success(_) => Ok(String::from(
-                response.get_result().unwrap()["signature"]
-                    .as_str()
-                    .unwrap(),
-            )),
-            JsonRpc::Error(_) => Err(HolochainError::ErrorGeneric(
-                serde_json::to_string(&response.get_error().unwrap()).unwrap(),
-            )),
-            _ => Err(HolochainError::ErrorGeneric("Signing failed".to_string())),
-        }
+        self.conductor_api.sign(payload)
```

Comment:
Is Context::sign used anywhere now?

Reply:
Yes, in the ribosome, i.e. inside core functions that still receive `Context` as a way to access an instance's resources.